packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -66,6 +66,13 @@ export const optionalToUint8Array = (array: Uint8Array | []): Uint8Array => {
   return Array.isArray(array) ? Uint8Array.from(array) : array;
 };
 
+interface LocalMLSServiceConfig extends MLSServiceConfig {
+  /**
+   * minimum number of key packages client should have available (configured to half of nbKeyPackages)
+   */
+  minRequiredNumberOfAvailableKeyPackages: number;
+}
```

Comment:
This is the config used by MLSService, it contains field that are not configurable by consumer (we set them internally).